sdk/kfserving/api/kf_serving_client.py

```diff
@@ -22,11 +22,11 @@
 class KFServingClient(object):
     """KFServing Apis."""
 
-    def __init__(self):
+    def __init__(self, kube_config=None):
```

Comment:
its not clear if this is the path or the config, why not just support `client_configuration` and `config_file` to pass through to load_kube_config?

Reply:
@rakelkar Thanks for your comments. Updated, for now `KFServingClient` support more, not only `config_file`, but also all parameters of `kubernetes.config.load_kube_config`.